webhook_to_fedora_messaging/models/service.py

```diff
@@ -7,12 +7,15 @@
 
 from webhook_to_fedora_messaging.database import Base
 
+from uuid import uuid4
+
 
 class Service(Base, UUIDCreatableMixin, CreatableMixin):
     __tablename__ = "services"
 
     id = Column(Integer, primary_key=True, nullable=False)
     user_id = Column(Integer, ForeignKey("users.id", ondelete="CASCADE"), unique=False, nullable=False)
+    token = Column(UnicodeText, unique=True, nullable=False, default=uuid4().hex)
```

Comment:
I don't think that's *really* necessary to make this token column unique, because services are identified by their `uuid` column. That said, it's not a blocker.

Reply:
The likeliness of us getting a collision on the server side is small but never zero. So that is indeed a valid point that keeps us from getting errors due to weird conditions introduced for the `token` column. I added the suggested change.